metaci/notification/forms.py

```diff
@@ -122,6 +123,43 @@ class Meta:
         widgets = {'user': forms.HiddenInput()}
 
 
+class AddPlanRepositoryNotificationForm(forms.ModelForm):
+
+    def __init__(self, *args, **kwargs):
+        super(AddPlanRepositoryNotificationForm, self).__init__(*args, **kwargs)
```

Comment:
these form helpers are very repetitive for the different types, is there any way we can pull out some of it into a superclass for the different notifications?

Reply:
In my view the entire notifications system needs an overhaul. To the user there should just be "notifications" and not several different types. This is a significant change, and because this is an urgent fix I opted to follow the established pattern for now. We can do it better if/when we overhaul the notifications.